main/OpenCover.Profiler/OpenCover.Profiler.vcxproj

```diff
@@ -364,6 +372,11 @@
   <ItemGroup>
     <Midl Include="OpenCoverProfiler.idl" />
   </ItemGroup>
+  <ItemGroup>

```

Comment:
There should be no need to introduce a new project for logging if you use ATLTRACE (or RELTRACE)